src/helpers/globalCss.ts

```diff
@@ -50,7 +50,493 @@ export const core = css`
     to {
         opacity: 1;
     }
-}`;
+}
+    
+:root {
+  /* BASE-COLORS */
+  --mdhui-grey-1-base: rgb(251, 252, 252);
```

Comment:
I'm curious why we have the base variables?  What does having a "base" color variable and then a 1:1 mapping with the actual color variables do for us?

Reply:
It's for enabling dark/light mode when we want to do that. The non "base" color variable gets assigned with whatever the current color palette is.